component-tools/src/main/java/org/talend/sdk/component/tools/ComponentValidator.java

```diff
@@ -183,25 +178,20 @@ private String validateIcon(final Icon annotation, final Collection<String> erro
                     prefixes.add(s + File.separator + ICONS + "dark" + File.separator + icon);
                 });
                 svgs = prefixes.stream().map(s -> new File(s + ".svg")).collect(toSet());
-                pngs = prefixes.stream().map(s -> new File(s + "_icon32.png")).collect(toSet());
             }
 
-            svgs.stream()
-                    .filter(f -> !f.exists())
-                    .forEach(
-                            svg -> log.error("No '" + stripPath(svg)
-                                    + "' found, this will run in degraded mode in Talend Cloud"));
             if (configuration.isValidateSvg()) {
                 errors.addAll(svgs.stream().filter(File::exists).flatMap(this::validateSvg).collect(toSet()));
             }
-            List<File> missingPngs = pngs.stream().filter(f -> !f.exists()).collect(toList());
-            if (!missingPngs.isEmpty()) {
-                errors.addAll(missingPngs.stream()
+
+            List<File> missingSvgs = svgs.stream().filter(f -> !f.exists()).collect(toList());
+            if (!missingSvgs.isEmpty()) {
+                errors.addAll(missingSvgs.stream()
                         .map(p -> String.format(
-                                "No icon: '%s' found, did you create - or generated with svg2png in resources?",
+                                "No '%s' found.",
                                 stripPath(p)))
                         .collect(toList()));
-                return "Missing icon(s) in resources.";
+                return "Missing icon(s) in resources:";
```

Comment:
why do we put ":" at the end?

Reply:
Because if this message shown, means that some icons were missing, next will list some missed icons, so here is ":" , means following list some icons.